### ISSUE TITLE
Please add action "remove kernels with non existing vmlinuz"

### ISSUE THREAD

**Issue by evsh** [mgorny/eclean-kernel2#3]
eclean-kernel had the mentioned mode. It was possible to remove unneeded kernel files manually and use eclean-kernel for cleaning. Possible use cases: LTS kernel and current kernel installed simultaneously; bisecting kernel, which produces different versions. It would be useful in these cases to be able to select which kernels to remove, either by deleting vmlinux files manually, or providing versions to `eclean-kernel -l` , e.g. `eclean-kernel --keep-build-dir 4.6.3-gentoo 4.4.6-gentoo `

**Comment by GreatEmerald** [comment]
Indeed; actually, eclean-kernel (1) used to by default remove all kernels not referenced by the bootloader, whereas eclean-kernel2 only allows keeping a set number of newest kernels.

**Comment by mgorny** [comment]
Different removal modes are work-in-progress. Can't promise when I'll have time to finish it though.
